04-cs-fundamentals/classroom/maths-exercises/essential-maths-exercises.md

```diff
@@ -0,0 +1,282 @@
+# Essential Mathematics - practice problems
+_CS fundamentals 0_
+
+## Overview
+Complete each problem manually (i.e. using paper and pencil) and check your solution with the one provided for each problem below.
+
+### Sorted items
+For each of the following list of objects, share whether the objects are: _sorted in ascending order_, _sorted in descending order_, _unsorted_.
+
+1.
+[**2, 4, 6, 8, 10**]
+<details><summary>click here to check your answer</summary>
+These numbers are sorted in ascending order.
```

Comment:
Again, you can get the markdown formatter to automatically indent and figure out numbering by indenting all of these by 4 spaces.

Reply:
Thanks, Dan! That definitely is the right way to go.